src/EnergyPlus/UnitarySystem.cc

```diff
@@ -1088,28 +1088,11 @@ namespace UnitarySystems {
 
         if (this->m_MySetPointCheckFlag) {
             if (!state.dataGlobal->SysSizingCalc && state.dataHVACGlobal->DoSetPointTest) {
-
-                if (this->m_CoolCoilExists) {
-                    int ControlNode = this->m_SystemCoolControlNodeNum;
-                    if (ControlNode > 0) {
-                        this->checkNodeSetPoint(state, AirLoopNum, ControlNode, CoolingCoil, OAUCoilOutTemp);
-                    }
-                }
-
-                if (this->m_HeatCoilExists) {
-                    int ControlNode = this->m_SystemHeatControlNodeNum;
-                    if (ControlNode > 0) {
-                        this->checkNodeSetPoint(state, AirLoopNum, ControlNode, HeatingCoil, OAUCoilOutTemp);
-                    }
-                }
-
-                if (this->m_SuppCoilExists) {
-                    int ControlNode = this->m_SuppHeatControlNodeNum;
-                    if (ControlNode > 0) {
-                        this->checkNodeSetPoint(state, AirLoopNum, ControlNode, SuppHeatCoil, OAUCoilOutTemp);
-                    }
-                }
-
+                bool e = false;
+                if (this->m_CoolCoilExists) e = this->checkNodeSetPoint(state, AirLoopNum, this->CoolCtrlNode, CoolingCoil, OAUCoilOutTemp);
+                if (this->m_HeatCoilExists) e = this->checkNodeSetPoint(state, AirLoopNum, this->HeatCtrlNode, HeatingCoil, OAUCoilOutTemp) || e;
```

Comment:
`if (x) e = something_long || e;`

could use the or/equals shorthand:

`if (x) e |= something_long`

which could be nicer than the trailing `|| e` on each line. No need to change now, just noting it's an option.

Reply:
That's the syntax I couldn't recall. I'll clean that up later.